integration/localnet/builder/bootstrap.go

```diff
@@ -85,6 +86,7 @@ func init() {
 	flag.IntVar(&testExecutionCount, "test-execution", DefaultTestExecutionCount, "number of test execution")
 	flag.UintVar(&nClusters, "nclusters", DefaultNClusters, "number of collector clusters")
 	flag.Uint64Var(&numViewsEpoch, "epoch-length", 10000, "number of views in epoch")
+	flag.Uint64Var(&numViewsPerSecond, "epoch-view-rate", 1, "number of views per second")
```

Comment:
I would suggest to call this `target-view-rate` because its describes a property of the network across all epochs (in comparison,  the other parameters `epoch-*` usually specify something for _one_ epoch, like the views within one epoch)